### PR TITLE
Add roblox extension

### PR DESCRIPTION
**Pull request by iamEvanYT**
## Description

This extension lets you fetch information about users & games on Roblox easily.

## Screencast

game-player-count

https://github.com/user-attachments/assets/8de906ee-7c4f-44b7-85a6-4f8e538269c8

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/roblox/package.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    }
+  ],
+  "preferences": [
+    {
+      "name": "userId",
+      "title": "Your User ID",
+      "description": "Your Roblox User ID",
+      "type": "textfield",
+      "placeholder": "User ID",
+      "required": false
```
Comment (pernielsentikaer) [inline extensions/roblox/package.json:90]:
```suggestion
      "required": true
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Can you record a screencast of this extension so I'm able to see what it can since I don't have Roblox user id or know anything about it

File: extensions/roblox/package.json (hunk 1 above)
Comment (iamEvanYT) [reply extensions/roblox/package.json:90]: This should not be required, as it is only required for one command. The other commands can function without it.

**Comment by iamEvanYT** [comment]
> Can you record a screencast of this extension so I'm able to see what it can since I don't have Roblox user id or know anything about it

Sure. This extension basically lets you fetch information about users & games.

https://github.com/user-attachments/assets/a1492ee5-2b23-48c2-9ddd-56faf5e7b2b5

File: extensions/roblox/package.json (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/roblox/package.json:127]: You can set `preferences:[]` directly for the given command instead of having them available for all commands

**Comment by iamEvanYT** [comment]
> Is this ready for another review?

Yep!

**Comment by iamEvanYT** [comment]
> I don't use Roblox but there's another extension in the Store called "Roblox" - https://www.raycast.com/TaylorRobinson/roblox-games.
> 
> Does it not make sense to add your features to the old one? Again, I'm not a Roblox user so I don't really know if it's doable. @iamEvanYT

@xmok 

First, that extension is called 'roblox-games', which is not what this extension does.
This extension can let you look at data about Users & Games, favourite games and more. I will also extend this extension with more features about Roblox if they are useful enough.

Secondly, the other extension is a bit old (2 years), and uses authentication which requires the Roblox App installed, and uses legacy APIs that might not be compatable.

TLDR: they are similar but different.

**Comment by iamEvanYT** [comment]
> Is this ready for another review?

@pernielsentikaer 

Yep! I have included some new screenshots and a screen recording on the main issue with everything new.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by iamEvanYT** [comment]
Last Commit 😄

Adds a command that uses Roblox Logs file to determine what game the player is on, and grab some information from there! 
Raycast Screenshot

**Comment by iamEvanYT** [comment]
Last Commit 😄

Adds a command that uses Roblox Logs file to determine what game the player is on, and grab some information from there! 
Raycast Screenshot
